mava/environment_loop.py

```diff
@@ -245,6 +248,72 @@ def should_run_loop(eval_interval_condition: Tuple) -> bool:
             )
             evaluation_duration = eval_duration_condition[1]
 
+        def run_evaluation(results: Any) -> None:
+            """Calculate the absolute metric"""
+
+            if self._executor.has(BaseNormalisation) and (
+                self._executor.store.global_config.normalise_observations
+                or self._executor.store.global_config.normalise_target_values
+            ):
+                warnings.warn(
+                    """The calculation of the absolute metric in
+                the case of normalisation is not supported"""
```

Comment:
Does the absolute metric not work when normalising for the same reason as the checkpointer? ie. We cannot effectively keep track of the normalisation params?

Reply:
The issue it will take time to investigate how to track the normalized params since they are not the same format as the usual one

Reply:
Created an [issue](https://github.com/instadeepai/Mava/issues/859) for it.